src/providers/https.ts

```diff
@@ -262,18 +265,21 @@ export interface CallableContext {
 }
 
 // The allowed interface for an http request for a callable function.
```

Comment:
I know we're hiding this, but just to document it better in the source code, can you revise this to:

The allowed interface for an HTTP request to a Callable function.

Reply:
Done.